src/backend/marsha/core/tests/test_utils_medialive_utils.py

```diff
@@ -4,6 +4,7 @@
 from django.test import TestCase, override_settings
```

Comment:
In your commit message `s/A test were incomplete/The test test_create_mediapackage_harvest_job was incomplete/`